encoding/descaling.md

```diff
@@ -1,477 +1,365 @@
-You can't descale to 1080p
-if the frame is already 1080p.
-If you have reason to believe
-that your show might be native 1080p,
-you've got to go with your gut.
-
-![Date A Live III — 01 (getnative graph)](images/descale_native1080_graph.png)
-
-An example of a graph for a native 1080p show.
-
-You may notice that the line swerves a bit
-in the first graph.
-There are going to be cases where
-you will get odd graphs like these,
-so it's important to know
-when the results are safe enough to descale
-or when they're too risky.
-Here is an example of
-a "bad" graph:
-
-![Miru Tights — 02 (getnative graph)](images/descale_bad_graph1.png)
+descaled = lvf.test_descale(clip, height=720, kernel='bicubic', b=1/3, c=1/3)
 
-```
-Output:
-Kernel: bicubic AR: 1.78 B: 0.33 C: 0.33
-Native resolution(s) (best guess): 869p, 848p
+comp = lvf.compare(clip, descaled)
+comp.set_output()
 ```
 
-The script has determined that
-it's likely either 848p
-or 869p.
-However,
-there are no clear spikes in this graph
-like there was in the Manaria Friends one.
-The results are not clear enough
-to work off of.
-Here's another example:
+The most reliable way to check for differences in vsedit is
+by zooming in 4x with Nearest Neighbor
+and switching between the frames
+with the left and right arrow buttons.
+You can enable the zoom here:
 
-![Black Lagoon (getnative graph)](images/descale_bad_graph2.png)
+![The zoom bar](images/descale/descale_zoom_a.png)
 
-```
-Output:
-Kernel: bicubic AR: 1.78 B: 0.33 C: 0.33
-Native resolution(s) (best guess): 1000p, 974p, 810p
-```
+Click "No zoom" and select "Fixed ratio".
+You can set the zoom level next to it,
+and the scaler used next to that.
+Make sure to set it to Nearest!
 
-This graph has a lot of unnatural swerves
-and it's impossible to determine
-what the native resolution is.
+Figuring out what to look out for
+is a case of trial and error.
+The best thing to keep an eye on
+is lineart as well as small detail like stars.
+It isn't uncommon for noise to become weaker after a rescale
+(since that's usually added after upscaling, too),
+so try not to be too distracted by that.
 
-Another pitfall you've got
-to watch out for
-is checking the results of a frame
-with letterboxing.
+[descale]: https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale
+[descale.py]: https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale/blob/master/descale.py
+[fmtc]: https://forum.doom9.org/showthread.php?t=166504
+[lvsfunc]: https://github.com/Irrational-Encoding-Wizardry/lvsfunc
 
-![Kizumonogatari I](images/descale_ararararagi.png)
 
-![Kizumonogatari I (getnative graph)](images/descale_ararararagi_graph.png)
+# Re-scaling
 
-You will have to crop them beforehand
-or they will return odd graphs like this.
+A fairly common practice is
+to upscale the descaled clip to a standard resolution
+(if it isn't already one, like for example 720p).
+The following scalers are generally recommended for this task.
```

Comment:
These seems like there's going to be a list after this line, so maybe needs some rewording.